library/src/main/kotlin/org/codice/compliance/SAMLSpecRefMessage.kt

```diff
@@ -19,378 +19,378 @@ package org.codice.compliance
+            """identifier uniqueness."""),
 
-    SAMLCore_3_2_2_b("If the response is not generated in response to a request, or if the ID attribute value of a " +
-            "request cannot be determined (for example, the request is malformed), then this attribute MUST NOT be " +
-            "present. Otherwise, it MUST be present and its value MUST match the value of the corresponding " +
-            "request's ID attribute."),
+    SAMLCore_3_2_2_b("""If the response is not generated in response to a request, or if the ID attribute value of a """ +
+            """request cannot be determined (for example, the request is malformed), then this attribute MUST NOT be """ +
+            """present. Otherwise, it MUST be present and its value MUST match the value of the corresponding """ +
+            """request's ID attribute."""),
 
-    SAMLCore_3_2_2_c("The identifier for the version of SAML defined in this specification is '2.0'."),
+    SAMLCore_3_2_2_c("""The identifier for the version of SAML defined in this specification is '2.0'."""),
 
-    SAMLCore_3_2_2_d("The time value is encoded in UTC, as described in Section 1.3.3."),
+    SAMLCore_3_2_2_d("""The time value is encoded in UTC, as described in Section 1.3.3."""),
 
-    SAMLCore_3_2_2_e("If it is present, the actual recipient MUST check that the URI reference identifies the " +
-            "location at which the message was received. If it does not, the response MUST be discarded. Some " +
-            "protocol bindings may require the use of this attribute (see [SAMLBind])."),
+    SAMLCore_3_2_2_e("""If it is present, the actual recipient MUST check that the URI reference identifies the """ +
+            """location at which the message was received. If it does not, the response MUST be discarded. Some """ +
+            """protocol bindings may require the use of this attribute (see [SAMLBind])."""),
 
-    SAMLCore_3_2_2_2("The value of the topmost <StatusCode> element MUST be from the top-level list provided in this " +
-            "section.\n" +
-            "The permissible top-level <StatusCode> values are as follows: \n " +
-            "- urn:oasis:names:tc:SAML:2.0:status:Success \n " +
-            "- urn:oasis:names:tc:SAML:2.0:status:Requester \n " +
-            "- urn:oasis:names:tc:SAML:2.0:status:Responder \n " +
-            "- urn:oasis:names:tc:SAML:2.0:status:VersionMismatch"),
+    SAMLCore_3_2_2_2("""The value of the topmost <StatusCode> element MUST be from the top-level list provided in this """ +
+            """section.""" +
+            "\n" + """The permissible top-level <StatusCode> values are as follows:""" +
+            "\n" + """- urn:oasis:names:tc:SAML:2.0:status:Success""" +
+            "\n" + """- urn:oasis:names:tc:SAML:2.0:status:Requester""" +
+            "\n" + """- urn:oasis:names:tc:SAML:2.0:status:Responder""" +
+            "\n" + """- urn:oasis:names:tc:SAML:2.0:status:VersionMismatch"""),
```

Comment:
✏️ This is nitpicky, but it seems like we could make an exception for cases like this where we don't need to escape any characters but we do need to be able to put in a newline.

Reply:
Agreed. I'd suggested this yesterday and now that I'm seeing it I immediately regret that. 😄